metrics/scaling/k8s_parallel.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/bash
```

Comment:
I think we should either combined this with k8s_scale.sh or extract more common tooling to share, one of the scripts is going to get stale quickly. But we can start this way.

Reply:
I agree. My gut says share the common code via libs, depending on just how similar tests are. A balancing act.
Yeah, I pulled out the api code in this PR, and I expect we will pull out and factor our/share more code as we add more tests.